packages/ts/react-crud/src/autocrud.tsx

```diff
@@ -84,7 +93,7 @@ export function ExperimentalAutoCrud<TItem>({ service, model, noDelete }: AutoCr
   return (
     <>
       <AutoCrudContext.Provider value={{ noDelete, editItem, deleteItem }}>
-        <div className="auto-crud">
+        <div className={`auto-crud ${className}`} id={id} style={style}>
```

Comment:
Please add a test to verify that the props are propagated to the rendered element.